tf2onnx/tfonnx.py

```diff
@@ -1664,21 +1664,35 @@ def floormod_op(ctx, node, name, args):
     return nodes
 
 
-def all_op(ctx, node, name, args):
+def reduce_logic_op(ctx, node, name, args):
     # T output = All(T x, list(int) reduce_indices, @bool keepdims)
+    # T output = Any(T x, list(int) reduce_indices, @bool keepdims)
     reduce_dim = node.inputs[1].get_tensor_value()
     if isinstance(reduce_dim, np.ndarray):
         reduce_dim = reduce_dim.tolist()
     else:
         reduce_dim = [reduce_dim[0]]
+    for i in reduce_dim:
+        utils.make_sure(i >= 0, "negative reduce axis is not supported in onnx for now")
 
-    utils.make_sure(all(i >= 0 for i in reduce_dim), "negative axis is not supported in onnx")
     cast = ctx.make_node(op_type="Cast", inputs=[node.input[0]], attr={"to": onnx_pb.TensorProto.INT32})
     keepdims = helper.get_attribute_value(node.get_attr("keep_dims"))
-    reduce_min = ctx.make_node(op_type="ReduceMin", inputs=cast.output, attr={"axes": reduce_dim, "keepdims": keepdims})
-    res = ctx.make_node(op_type="Cast", inputs=reduce_min.output, attr={"to": onnx_pb.TensorProto.BOOL},
+    op_type = "ReduceMin" if node.type == "All" else "ReduceSum"
```

Comment:
you can put ReduceMin in the mapping 

_OPSET_6 = { 
    "All": (reduce_logic_op, ["ReduceMin"]),

Reply:
maybe ["All", "ReduceMin"], i knew the mechanism while i thought the op's conversion logic is not complex, so it's better to put it in function so reader not needs to figure out what is "args"

Reply:
dont' think so. you can check Max/Min/Mean's implementation. 

and have you ever consider use existing reduce_op to do the implementation?